.github/workflows/check-pr-title.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Check PR Title'
+on: 
+  pull_request:
+    types: [edited, opened, synchronize, reopened]
+
+jobs:
+  title-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: naveenk1223/action-pr-title@master
+        with:
+          regex: '[a-zA-Z]+\:.+' # Regex the title should match.
+          allowed_prefixes: 'ENH,FIX,STYLE,DOC,DEL' # title should start with the given prefix
+          prefix_case_sensitive: true # title prefix are case insensitive
+          min_length: 10 # Min length of the title
+          max_length: 100 # Max length of the title
```

Comment:
From [How to write a commit message that will make your mom proud](https://robertcooper.me/post/git-commit-messages):

> To prevent a commit message's subject to be cut off in GitHub, make sure to limit the character length to 72 characters.